app/helpers/blacklight/catalog_helper_behavior.rb

```diff
@@ -289,6 +289,24 @@ def render_search_to_page_title_filter(facet, values)
     t('blacklight.search.page_title.constraint', label: filter_label, value: filter_value)
   end
 
+  def render_search_title(params)
+    unless t('blacklight.search.page_title.title', default: '').blank?
+      Deprecation.warn(
+        Blacklight::CatalogHelperBehavior,
+        'The blacklight.search.page_title.title i18n key has been deprecated; use blacklight.search.page_title.with_constraints and blacklight.search.page_title.without_constraints instead.'
+      )
+      return t('blacklight.search.page_title.title',
+               constraints: render_search_to_page_title(params),
+               application_name: application_name)
+    end
+
+    return t('blacklight.search.page_title.without_constraints') unless query_has_constraints?(params)
+
+    t('blacklight.search.page_title.with_constraints',
+      :constraints => render_search_to_page_title(params),
+      :application_name => application_name)
+  end
+
   def render_search_to_page_title(params)
```

Comment:
This method should really be called something like `render_search_constraints`, but renaming it would have to wait for a major version update.  Should I add a deprecation notice now?